components/Tabs/README.md

```diff
@@ -116,6 +116,76 @@ navigation bar. The bar will automatically update with the appropriate styling.
 
 - - -
 
+### Theming
+
+You can theme a TabBar with your app's color scheme and typography scheme using the ColorThemer and TypographyThemer extensions.
+
+You must first add the Themer extensions to your project:
+
+``` bash
+pod 'MaterialComponents/Tabs+Extensions/ColorThemer'
+pod 'MaterialComponents/Tabs+Extensions/TypographyThemer'
+```
+
+You can then import the theming APIs:
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+``` swift
+import MaterialComponents.MaterialTabs_ColorThemer
+import MaterialComponents.MaterialTabs_TypographyThemer
+```
+
+#### Objective-C
+
+``` objc
+#import "MaterialTabs+ColorThemer.h"
+#import "MaterialTabs+TypographyThemer.h"
+```
+<!--</div>-->
+
+MDCTabBarColorThemer allows you to theme a tab bar with your app's color scheme. 
+
+MDCTabBarTypographyThemer allows you to theme a tab bar with your app's typography scheme. 
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+``` swift
+let colorScheme = MDCSemanticColorScheme()
+let typographyScheme = MDCTypographyScheme()
+
+let tabBar = MDCTabBar()
+
+// Apply regular style
+MDCTabBarColorThemer.apply(colorScheme, toTabs: tabBar)
```

Comment:
I think this has to be applySemanticColorScheme

Reply:
fixed straight to dev since this was already pushed.